src/metabase/config.clj

```diff
@@ -84,9 +84,13 @@
 (defn config-bool "Fetch a configuration key and parse it as a boolean."  ^Boolean [k] (some-> k config-str Boolean/parseBoolean))
 (defn config-kw   "Fetch a configuration key and parse it as a keyword."  ^Keyword [k] (some-> k config-str keyword))
 
-(def ^Boolean is-dev?  "Are we running in `dev` mode (i.e. in a REPL or via `clojure -M:run`)?" (= :dev  (config-kw :mb-run-mode)))
-(def ^Boolean is-prod? "Are we running in `prod` mode (i.e. from a JAR)?"                       (= :prod (config-kw :mb-run-mode)))
-(def ^Boolean is-test? "Are we running in `test` mode (i.e. via `clojure -X:test`)?"            (= :test (config-kw :mb-run-mode)))
+(def run-mode
```

Comment:
```suggestion
(def ^:private run-mode
```

A big benefit of `is-{dev/prod/test}?` is that we don't have to worry about whether the run mode is spelled `:dev` or `:development`, etc. It'd be nice to keep the specific keywords used locked up in this ns